pydash/pydash_app/dashboard/aggregator/statistics.py

```diff
@@ -171,29 +172,29 @@ def add_together(self, other, dependencies_self, dependencies_other):
         return aet
 
 
-class VisitsPerDay(Statistic):
-    def should_be_rendered(self):
-        return True
-
-    def empty(self):
-        return defaultdict(int)
-
-    def field_name(self):
-        return 'visits_per_day'
-
-    def perform_append(self, endpoint_call, dependencies):
-        date = endpoint_call.time.date()
-        self.value[date] += 1
-
-    def rendered_value(self):
-        return date_dict(self.value)
-
-    def add_together(self, other, dependencies_self, dependencies_other):
-        vpd = VisitsPerDay()
-        keyset = set(self.value.keys()).union(set(other.value.keys()))
-        for key in keyset:
-            vpd.value[key] = self.value[key] + other.value[key]
-        return vpd
+# class VisitsPerDay(Statistic):
```

Comment:
I assume this will be added back later?

Reply:
Right, I should make a comment about why I comment out stuff.
I commented this one out, as `aggregator_group.fetch_aggregators_per_timeslice` facilitates this and more, so keeping this as an entire statistic seems rather redundant and wasteful of memory.

Reply:
Yes, makes sense. But rather than commenting it out, remove it (we can always go back into git history if it turns out we need it :smiley:)